tests/index.spec.ts

```diff
@@ -328,3 +328,66 @@ describe('w/ hooks', () => {
     expect(store.after).toEqual('after')
   })
 })
+
+describe('w/ serializer', () => {
+  it('Initializes Correctly', async () => {
+    //* arrange
+    const initial = { lorem: 'ipsum' }
+    initializeLocalStorage(key, initial)
+
+    const deserializer = vi.fn<[string], StateTree>(value => JSON.parse(value))
+
+    const useStore = defineStore(key, {
+      state: () => ({
+        lorem: '',
+      }),
+      persist: {
+        serialize: {
+          serialize: JSON.stringify,
+          deserialize: deserializer,
+        },
+      },
+    })
+
+    //* act
+    await nextTick()
+    const store = useStore()
+
+    //* assert
+    expect(store.lorem).toEqual('ipsum')
```

Comment:
If I remove this line `store` will be unused, fixing that `useStore` will be unused